site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -367,7 +367,7 @@ To **resolve** this issue you will need to set `allowPrivilegeEscalation` to `tr
 
 ##### NGINX Plus failure to start or traffic interruptions
 
-Beginning with NGINX Gateway Fabric 1.5.1, NGINX Plus requires a valid JSON Web Token (JWT) to run. If this is not set up properly, or your JWT token has expired, you may see errors in the NGINX logs that look like the following:
```

Comment:
@sjberman also updated this one, since you caught it the PR to main

Reply:
LGTM

Reply:
How come these files are getting missed? these are auto-generated, right?